ansible/playbooks/roles/repository/files/download-requirements/requirements/aarch64/cranes.yml

```diff
@@ -1,4 +1,6 @@
 ---
 cranes:
-  'https://github.com/google/go-containerregistry/releases/download/v0.11.0/go-containerregistry_Linux_arm64.tar.gz':
-    sha256: 84653ec8297389ded927f120ea5bc2703423046f64c56b538197875f81ba4cd6
+  go-containerregistry:
+    primary:
```

Comment:
How about using a list instead of dictionary for shorter structure?
It would even allow to have more than two URLs.

Example:
 ```
go-containerregistry:
  alternatives:
    - url: <1st>
      sha256: <1st_sha>
    - url: <2nd>
      sha256: <2nd_sha>
    - url: <3rd>
      sha256: <3rd_sha>
```